libs/catalog/src/lib/catalog.module.ts

```diff
@@ -0,0 +1,12 @@
+import { NgModule } from '@angular/core'
+import { SiteTitleComponent } from './site-title/site-title.component'
+import { UiModule } from '../../../ui/src'
```

Comment:
must be aliases like `@lib/gn-ui`

Reply:
`import { UiModule } from '@lib/ui'`works. Would it make sense to prefix lib alias with `gn-` ?
There is no Intellij shortcut to properly write imports? or npm build task to clean it up?

Reply:
If everything is well configured
- alias in tsconfig
- classes exported in lib index.ts

Then Intellij do the correct import with the alias
If Intellij does not, it tells that there is mistake, fix it, remove the import and ask IntJ to redo the import, it should be correct then.

Note that Intellij sometimes uses alias to point to its own library, but path within the same library must be relatives.